recipes/chemper/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "chemper" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mobleylab/chemper/archive/1.0.0.tar.gz
```

Comment:
This should be `https://github.com/mobleylab/chemper/archive/{{ version }}.tar.gz`